util/csvresponse.py

```diff
@@ -112,7 +114,7 @@ def _create_filter_list(self, keys):
 
     @staticmethod
     def _is_qc_param(param):
-        return QC_EXECUTED in param or QC_RESULTS in param
+        return QC_EXECUTED in param or QC_RESULTS in param or QARTOD_PRIMARY in param or QARTOD_SECONDARY in param
```

Comment:
Another way I believe you could state this is (could make it cleaner):
return any(val in param for val in [QC_EXECUTED,QC_RESULTS,QARTOD_PRIMARY,QARTOD_SECONDARY])